tests/ComputeSharp.D2D1.Tests/ComputeSharp.D2D1.Tests.csproj

```diff
@@ -4,16 +4,17 @@
     <TargetFramework>net6.0</TargetFramework>
     <IsPackable>false</IsPackable>
     <NoWarn>$(NoWarn);CS0419;CA1416</NoWarn>
+    <LangVersion>preview</LangVersion>
```

Comment:
Please keep in mind that language features can be removed in the preview version.

Reply:
Ah, yeah, good point. I guess in this case I'm not really concerned, first because ComputeSharp is still in alpha anyway, and also because with the raw multiline string literals feature specifically, I'm confident enough that won't get removed at this point 🙂

Reply:
@Sergio0694 I'm curious though, why was changing the language version needed? It doesn't look like you're actually using raw string literals anywhere in this PR.

Reply:
The raw multiline string literals are in the generated code, so I enabled it here to ensure that (1) everything still built and run correctly and the generator didn't mess things up with the raw multiline strings, and also that I could check out the generated files in solution explorer and see that the expected syntax was actually being used 🙂

Reply:
Oh, the test project directly uses the source generators? 😮 That's interesting.

Reply:
Yeah this is how ComputeSharp works, it just _cannot_ work without the source generator. They're necessary to implement all the logic to transpile C# to HLSL and to generate other various methods to load and dispatch the shaders on the GPU. The interface you actually implement for a shader (eg. `ID2D1PixelShader`) actually has several methods that are just meant to be implemented by the generator. The library as a whole is just entirely built with source generators in mind, and it's completley unusable if they're not running 😄